### PR TITLE
mubert-base.gson.mid-261.idx-175127.17.mutant

### PR DESCRIPTION
**Pull request by jadshaker**

### CODE REVIEW

File: gson/gson/src/main/java/com/google/gson/internal/bind/JsonTreeWriter.java [hunk 1]
```diff
@@ -188,7 +188,6 @@ private void put(JsonElement value) {
       put(new JsonPrimitive(value));
       return this;
     }
```
Comment (faustinoaq) [inline gson/gson/src/main/java/com/google/gson/internal/bind/JsonTreeWriter.java:190]: Needs to fix indentation to match formatting

**Review by faustinoaq — CHANGES_REQUESTED** [review]

File: gson/gson/src/main/java/com/google/gson/internal/bind/JsonTreeWriter.java (hunk 1 above)
Comment (darko-marinov) [reply gson/gson/src/main/java/com/google/gson/internal/bind/JsonTreeWriter.java:190]: @jadshaker please address this in the next 2-3 days or we'll have to assign to someone else.

Comment (jadshaker) [reply gson/gson/src/main/java/com/google/gson/internal/bind/JsonTreeWriter.java:190]: Fixed

**Comment by darko-marinov** [comment]
The diff https://github.com/jadshaker/UIUCPlus/compare/461dfea3626452643d4138fef934535979fe87e9..mubert-base.gson.mid-261.idx-175127.17.mutant still looks unnecessarily big.

**Pull request merged**